content/hi/tightly-coupled-architecture.md

```diff
@@ -0,0 +1,10 @@
+---
+title: साइट विश्वसनीयता इंजीनियरिंग (Tightly Coupled Architecture)
```

Comment:
Hi can we please replace साइट विश्वसनीयता इंजीनियरिंग with टाइट्ली कपल्ड आर्किटेक्चर